src/store/customer/reducer.js

```diff
@@ -0,0 +1,12 @@
+import { SET_CUSTOMER } from './actions'
+
+const reducer = (state = {}, action) => {
+  switch (action.type) {
+    case SET_CUSTOMER:
```

Comment:
Besides setting the customer you should also have a possibility to remove him (after logout)